pure2.md

```diff
@@ -0,0 +1,535 @@
+
+The offline-update snapshot metadata lists version numbers and filenames of all offline-update targets metadata files.
+
+For each offline-update targets metadata file on the repository, the offline-update snapshot metadata SHALL contain the following information:
+
+1.  The filename and version number of the offline-update targets metadata file.
+
+### Offline-update targets metadata
+
+Each offline-update targets metadata file on a repository contains information about a set of images eligible to be installed on devices. Like other Director targets metadata files, offline-update targets metadata files SHALL NOT contain delegations.
+
+When a device is validating offline-update targets metadata, it will only execute an update if it can unambiguously determine which images to install. For this reason, the metadata SHOULD include a mechanism to permit disambiguation in multi-ECU systems. See the deployment considerations section for more detail on this recommendation.
+
+### Addendum to Director repository snapshot metadata requirements
+
+Offline-update targets metadata files SHOULD NOT be listed in the main snapshots metadata file of the Director repository, if implemented. Because the offline-update role signs its own snapshot metadata, listing those files in the main snapshots metadata file is redundant, and increases the size of a file that is downloaded very frequently.
+
+### Required files for offline updates
+
+The following elements SHALL be present on any offline-update storage location (in a "well-known" location configured in advance on the on the Primary):
+
+-   Image repository metadata:
+
+    -   All Root metadata files from the Image repository, from version 1 to the most-recent version at the time the offline update is generated
+
+    -   The latest version of every Targets metadata file, including delegations, at the time the offline update is generated
+
+    -   The latest version of the Snapshot metadata file at the time the offline update was generated
+
+-   Director repository metadata:
+
+    -   All Root metadata files from the Director repository, from version 1 to the most-recent version at the time the offline update is generated
+
+    -   The offline-update Snapshot metadata file
+
+    -   Exactly one offline-update targets metadata file
+
+-   All target images referenced in the offline-update targets metadata file
+
+### Metadata verification procedures
+
+The procedures for verifying offline-update metadata are similar to the procedures for verifying normal director metadata. However, there are some key differences. The procedures for sending images from the primary to secondaries, as well as for verifying and installing the images once the metadata has been verified, remain the same.
+
+Note: wherever the "Uptane Standard" or "the Standard" is referenced, it refers to version 1.2.0 of the Standard, available at https://uptane.github.io/uptane-standard/1.2.0/uptane-standard.html.
```

Comment:
Does this mean we will need to update this once v2 is released? How do we handle that going forward?

Reply:
Good question. This note might not even be necessary.

Reply:
If v2 comes out before this PURE is accepted, we can update this here. Otherwise once this is accepted and included in the Standard I think it's fine to let the PURE reference an old version of the Standard.

Reply:
I think it's not a bad idea to reference the specific version the PURE was written in mind for. That way if there is any ambiguity, it's at least clear what version was intended, and if it needs to be updated later to resolve compatibility issues, it's easier to know what it needs to compare with.